packages/quantic/cypress/integration/standalone-search-box/standalone-search-box-expectations.ts

```diff
@@ -0,0 +1,53 @@
+import {should} from '../common-selectors';
+import {
+  StandaloneSearchBoxSelectors,
+  StandaloneSearchBoxSelector,
+} from './standalone-search-box-selectors';
+
+function standaloneSearchBoxExpectations(
+  selector: StandaloneSearchBoxSelector
+) {
+  return {
+    displayInputSearchBox: (display: boolean) => {
+      selector
+        .input()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the input search box`);
+    },
+    displaySearchIcon: (display: boolean) => {
+      selector
+        .searchIcon()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the search icon`);
+    },
+    displaySearchButton: (display: boolean) => {
+      selector
+        .searchButton()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the search button`);
+    },
+    displayCloseIcon: (display: boolean) => {
```

Comment:
I know technically the icon name is "close" but we're using it to "clear" the text content so I'm wondering if `displayClearIcon` or perhaps `displayClearButton` might make more sense here. What do you think?

Reply:
done